docker-compose.yaml

```diff
@@ -0,0 +1,38 @@
+# Base in-memory configuration for the registry service
+name: bytecodealliance-registry
+services:
+  operator-key:
+    container_name: operator-key
+    image: debian:bullseye-slim
+    # TODO: generate operator-key at runtime
+    command:
+      - /bin/bash
+      - -c
+      - >-
+        if [[ ! -f /var/secrets/warg/operator-key ]]; then
+          echo -n "ecdsa-p256:I+UlDo0HxyBBFeelhPPWmD+LnklOpqZDkrFP5VduASk=" >/var/secrets/warg/operator-key
+        fi
+    volumes:
+      - warg-secrets:/var/secrets/warg:rw
+  warg-registry:
+    build:
+      target: warg-server
+    ports:
+      - "8090"
+    environment:
+      - WARG_OPERATOR_KEY_FILE=/var/secrets/warg/operator-key
```

Comment:
In `k8s` this would exist as the value of key `operator-key` in a `Secret` mounted at `/var/secrets/warg`. It's probably good to check the different ways of configuring some options works.